mx.pwmc.Svgvi.json

```diff
@@ -0,0 +1,124 @@
+{
+  "app-id": "mx.pwmc.Svgvi",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.32",
+  "sdk": "org.gnome.Sdk",
+  "command": "mx.pwmc.Svgvi",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--filesystem=home",
```

Comment:
It looks like the only filesystem interaction is pretty simple though file open and save. Can you not switch the FileChooser calls to FileChooserNative and automatically get Portals support and remove the filesystem entirely? https://gitlab.com/search?utf8=%E2%9C%93&search=filechooser&group_id=&project_id=10414336&search_code=true&repository_ref=master doesn't look like you're that custom and https://developer.gnome.org/gtk3/stable/gtk3-GtkFileChooserNative.html should support that.

Reply:
I interpreted your request to use FileChooserNative, so I've switched to it.

About file system, are you suggesting to remove `"--filesystem=home" `, if that the case how do the application will get a file in the local or a network file system?

Reply:
I've tested locally and removing --filesystem=home while using Native file chooser, have no impact. So I've switched to them.
Updating the manifest.